src/gleam/dynamic.gleam

```diff
@@ -38,9 +42,9 @@ if erlang {
   ///    True
   ///
   ///    > bit_string(from(123))
-  ///    Error("Expected a BitString, got `123`")
+  ///    Error(DecoderError(expected: "bit_string", found: "int"))
```

Comment:
Do you think it might be better to use CamelCase? Typically types are written this way in Gleam.

Reply:
Having said that, plain English descriptions as with tuples may be clearer still... Hmm.

Reply:
I've switched to CamelCase and attempted to address some inconsistencies.